container.go

```diff
@@ -625,7 +625,7 @@ func (c *Container) kill(signal syscall.Signal, all bool) error {
 	}
 
 	if c.state.State != StateReady && c.state.State != StateRunning {
-		return fmt.Errorf("Container not ready or running, impossible to signal the container")
+		return Errorf("Container not ready or running, impossible to signal the container")
```

Comment:
This looks very good and returning better error messages will definitely help for faster debug. One question though, why is that only applied to this specific error ?

Reply:
Nevermind, I missed that it was only an example...